### DISCUSSION TITLE
Re-create Azure tags in existing azure environment

### DISCUSSION THREAD

**Discussion by gbalajicl** [Azure/ResourceModules#3385] [Q&A] [answered]
Hello All,

We are assigning tags to Azure resources.  Some of the tags are assigned using Azure policy, and some of the tags are assigned manually.  
We have assigned tags to resources by using the Azure policy effect "append".
Now we want to assign the same tags with the Policy effect "modify". 

Can you please suggest to me the best process for how to **re-create** tags using IaC Terraform? or **How to re-create** tags using policy? I need suggestions on this tagging **process**. What's the best approach to do it and how to do it?

Example tags: ApplicationName, Owner, CostCenter,

**Comment by eriqua** [comment, marked as answer]
Hi @gbalajicl, thanks for opening this discussion and sorry for the late reply. 

CARML evolved to and has been rebranded as the Bicep version of Azure Verified Modules ([AVM](https://aka.ms/AVM)). 
All CARML GitHub discussions are being converted to CARML GitHub issues, in order for the team to be able to further triage them and migrate them to AVM issues, if applicable.

Your question seems more related to Terraform, which the CARML initiative didn't cover back in the days. The good news is that AVM also covers Terraform DSL, so your question may find a good answer once migrated.

Please refer to issue #4547 going forward.
